src/features/surveys/l10n/messageIds.ts

```diff
@@ -106,6 +106,42 @@ export default makeMessages('feat.surveys', {
       ),
     },
   },
+  insights: {
+    export: {
+      errorNotFound: m(
+        'Error occurred while exporting: Did not find plot root.'
+      ),
+      errorUnknown: m('Unknown error occurred while exporting.'),
+      toPng: m('Export to png'),
+      toSvg: m('Export to svg'),
+    },
+    optionsFields: {
+      subheader: m<{
+        answerCount: number;
+        totalSelectedOptionsCount: number;
+      }>(
+        'In total, there were {answerCount} answers and {totalSelectedOptionsCount} selected options.'
+      ),
+      tabs: {
+        barPlot: m('Bar'),
+        piePlot: m('Pie'),
+      },
+    },
+    textFields: {
+      subheader: m<{
+        answerCount: number;
+        totalUniqueWordCount: number;
+        totalWordCount: number;
+      }>(
+        'In total, there were {answerCount} answers, {totalWordCount} words and {totalUniqueWordCount} unique words.'
```

Comment:
Does this maybe need pluralization? I reckon it will be unlikely to have just one word in a survey, so maybe it's not necessary?